src/Tokenizer/Analyzer/CommentsAnalyzer.php

```diff
@@ -194,6 +194,12 @@ private function isStructuralElement(Tokens $tokens, int $index): bool
             return true;
         }
 
+        if ($token->isGivenKind(T_CASE) && \defined('T_ENUM')) {
+            $caseParent = $tokens->getPrevTokenOfKind($index, [[T_ENUM], [T_SWITCH]]);
+
+            return null === $caseParent || $tokens[$caseParent]->isGivenKind([T_ENUM]);
```

Comment:
Can `$caseParent` ever be `null`? If, so let's have a test to cover such a case.

Reply:
I've added that based on `getPrevTokenOfKind()`'s return type, but in practice there's no chance for that 😅. I'll remove this condition.